### PR TITLE
Queue (mainly) style consistency

### PR DESCRIPTION
**Pull request by Wadeck**
- Removal of null check, while annotation + ecosystem search proves it's never occurring
- Small code style
- Cleanup of tab vs spaces

(tip: review the PR with "whitespace changes ignored")

🤷‍♂️ the class is so messy, I tried to keep the changes as small as possible, but there are so many stuff not consistent.

No ticket.

### Proposed changelog entries

* n/a

### Proposed upgrade guidelines

N/A

### Submitter checklist

- [n/a] (If applicable) Jira issue is well described
- [n/a] Changelog entries and upgrade guidelines are appropriate for the audience affected by the change (users or developer, depending on the change). [Examples](https://github.com/jenkins-infra/jenkins.io/blob/master/content/_data/changelogs/weekly.yml)
  * Fill-in the `Proposed changelog entries` section only if there are breaking changes or other changes which may require extra steps from users during the upgrade
- [n/a] Appropriate autotests or explanation to why this change has no tests
- [n/a] For dependency updates: links to external changelogs and, if possible, full diffs

### Maintainer checklist

Before the changes are marked as `ready-for-merge`: 

- [ ] There are at least 2 approvals for the pull request and no outstanding requests for change
- [ ] Conversations in the pull request are over OR it is explicit that a reviewer does not block the change
- [ ] Changelog entries in the PR title and/or `Proposed changelog entries` are correct
- [ ] Proper changelog labels are set so that the changelog can be generated automatically
- [ ] If the change needs additional upgrade steps from users, `upgrade-guide-needed` label is set and there is a `Proposed upgrade guidelines` section in the PR title. ([example](https://github.com/jenkinsci/jenkins/pull/4387))
- [ ] If it would make sense to backport the change to LTS, a Jira issue must exist, be a _Bug_ or _Improvement_, and be labeled as `lts-candidate` to be considered (see [query](https://issues.jenkins-ci.org/issues/?filter=12146)).

### CODE REVIEW

File: core/src/main/java/hudson/model/Queue.java [hunk 1]
```diff
@@ -810,17 +809,18 @@ private WaitingItem peek() {
...
         return items;
     }
 
     private List<Item> checkPermissionsAndAddToList(List<Item> r, Item t) {
-        if (t.task instanceof hudson.security.AccessControlled) {
-            if (((hudson.security.AccessControlled)t.task).hasPermission(hudson.model.Item.READ)
-                    || ((hudson.security.AccessControlled) t.task).hasPermission(hudson.security.Permission.READ)) {
+        if (t.task instanceof AccessControlled) {
+            AccessControlled taskAC = (AccessControlled) t.task;
+            if (taskAC.hasPermission(hudson.model.Item.READ)
```
Comment (timja) [inline core/src/main/java/hudson/model/Queue.java:822]: any reason hudson.model.Item isn't imported?

File: core/src/main/java/hudson/model/Queue.java [hunk 2]
```diff
@@ -810,17 +809,18 @@ private WaitingItem peek() {
...
+                    || taskAC.hasPermission(hudson.security.Permission.READ)) {
```
Comment (timja) [inline core/src/main/java/hudson/model/Queue.java:823]: same here for Permission

**Review by timja — APPROVED** [review]

File: core/src/main/java/hudson/model/Queue.java (hunk 1 above)
Comment (Wadeck) [reply core/src/main/java/hudson/model/Queue.java:822]: Due to Queue.Item defined in the class :)

File: core/src/main/java/hudson/model/Queue.java [hunk 3]
```diff
@@ -358,7 +358,6 @@ public LoadBalancer getLoadBalancer() {
     }
 
     public void setLoadBalancer(@NonNull LoadBalancer loadBalancer) {
-        if(loadBalancer==null)  throw new IllegalArgumentException();
```
Comment (Wadeck) [inline core/src/main/java/hudson/model/Queue.java:361]: I checked the ecosystem, everytime this method is called with a valid nonNull loadBalancer.

**Review by res0nance — APPROVED** [review]

**Pull request merged**